docs/guides/setup/installing/spark-emr.md

```diff
@@ -1,6 +1,48 @@
 ---
-title: Installing great_expectations in Spark EMR 
+title: Installing great_expectations in Spark EMR cluster
```

Comment:
Please make titles consistent with the specific language in the design document: https://github.com/superconductive/design/blob/main/docs/20210421_knoxpod_golden_paths_mental_map/toc.md#installation-this-still-needs-to-be-fleshed-out-and-finalized

Reply:
changed